.github/workflows/self-new-model-pr-caller.yml

```diff
@@ -0,0 +1,123 @@
+name: New model PR CI
+
+on:
+  pull_request:
```

Comment:
we could probably also trigger this on `merge to main event`, but the point of this PR (so far) is to make sure the new model PR has green tests for that model.